functions/__sf_section_node.fish

```diff
@@ -0,0 +1,31 @@
+function __sf_section_node
+  __sf_util_set_default SPACEFISH_NODE_SHOW true
+  __sf_util_set_default SPACEFISH_NODE_SYMBOL ⬢
+  __sf_util_set_default SPACEFISH_NODE_COLOR green
```

Comment:
You should add configuration comments like [this](https://github.com/matchai/spacefish/blob/master/functions/__sf_section_battery.fish#L6). You can look at the ones from spaceship